sympy/sets/sets.py

```diff
@@ -206,6 +206,12 @@ def _complement(self, other):
         elif isinstance(other, FiniteSet):
             return FiniteSet(*[el for el in other if self.contains(el) != True])
 
+    def _symmetricdifference(self, other):
+        """
+        This function should only be used internally
```

Comment:
I think, you can remove this docstring.  Method is private if it starts with underscore.